ci/dockerhub-login-step.yml

```diff
@@ -0,0 +1,7 @@
+steps:
+  - task: Docker@1
+    displayName: Dockerhub registry login
+    inputs:
+      command: login
+      containerregistrytype: Container Registry
+      dockerRegistryEndpoint: dockerhub
```

Comment:
where is this defined?

Reply:
This is defined under project settings on azure pipelines.